src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -144,7 +141,7 @@ const FluxEditorMonaco: FC<Props> = ({
       <ErrorBoundary>
         <div className={wrapperClassName} data-testid="flux-editor">
           <MonacoEditor
-            language={language}
+            language={FLUXLANGID}
```

Comment:
Why not using the enum `LanguageType.FLUX`? It's the same string `flux` as `FLUXLANGID`. Since now we have two supported languages and it will grow more in the future, I think it's better to use the same source of truth

Reply:
To trigger the syntax loading into the monaco-editor.

Reply:
Got it. I was confused with resource language type in [resources/index.ts](https://github.com/influxdata/ui/blob/master/src/dataExplorer/components/resources/index.ts#L6) with `FLUXLANGID` 😅

Reply:
Totally agreed. It's definitely not ideal. Which is why we should eventually change how/why we load the language syntax (for all languages), to make it more explicit.